### PR TITLE
In the case of Tuya end Moes, additional identification is carried out through the manufacturerName.

### PR DESCRIPTION
**Pull request by vladi1234**

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -746,6 +746,16 @@
         "url": "https://github.com/Koenkk/zigbee-OTA/raw/master/images/Ledvance/Undercabinet_TW_Z3_IM0046_00103101-encrypted_11_20_2018_Tue_101550_96_withoutMF.ota",
         "path": "images/Ledvance/Undercabinet_TW_Z3_IM0046_00103101-encrypted_11_20_2018_Tue_101550_96_withoutMF.ota"
     },
+    {
+        "fileVersion": 87,
+        "fileSize": 234174,
+        "manufacturerCode": 4098,
+        "manufacturerName": "_TZE200_ckud7u2l", "_TZE200_cwnjrr72",
```
Comment (Koenkk) [inline index.json:753]: This is invalid json syntax, change it to an array

Comment (vladi1234) [reply index.json:753]: Please help

Comment (vladi1234) [reply index.json:753]: "manufacturerName": ["_TZE200_ckud7u2l", "_TZE200_cwnjrr72"],

**Comment by Koenkk** [comment]
I will wait 2 months before merging this. The reason behind this is that people who are still using older z2m versions will get this OTA update even when the manufacturer name is different since those version do not check for the manufacturer name yet.

**Comment by Koenkk** [comment]
I will merge it on 1 November. (https://github.com/Koenkk/zigbee-OTA/pull/50#issuecomment-913736281)

**Comment by wizu** [comment]
@Koenkk friendly reminder ;)

**Comment by Koenkk** [comment]
@wizu yes, thanks!

**Comment by vladi1234** [comment]
@Koenkk Looks like everything is working.
Thanks!

**Pull request merged**